src/styles/StyleUtils.ts

```diff
@@ -146,12 +143,6 @@ const avatarSizes: Record<AvatarSizeName, AvatarSizeValue> = {
     [CONST.AVATAR_SIZE.SMALL_NORMAL]: variables.avatarSizeSmallNormal,
 };
 
-const emptyAvatarStyles: Record<EmptyAvatarSizeName, ViewStyle> = {
-    [CONST.AVATAR_SIZE.SMALL]: styles.emptyAvatarSmall,
-    [CONST.AVATAR_SIZE.MEDIUM]: styles.emptyAvatarMedium,
-    [CONST.AVATAR_SIZE.LARGE]: styles.emptyAvatarLarge,
-};
-
 const avatarFontSizes: Partial<Record<AvatarSizeName, number>> = {
```

Comment:
Could you remind me why we're making these changes to StyleUtils?

cc @chrispader

Reply:
It's because these reference `styles`, which is now just a function that needs the current theme. Theoretically we could hook this up to subscribe to the current theme in Onyx and keep a reference to the current styles outside of a hook, something like this:

```js
let currentTheme;
let currentStyles;

// do some stuff here to connect to the preferred theme and keep an up-to-date version of the styles object in currentStyles and use that
Onyx.connect()
```

But I don't think that's a good practice because the component still needs to use the hook or HOC to ensure that it re-renders when the theme changes.

Does that make sense?